app/javascript/react/containers/ReviewForm.js

```diff
@@ -8,14 +8,7 @@ class ReviewForm extends Component {
     this.state = {
       body: "",
       rating: "",
-      ratingOptions: [
-        "0 Stars",
-        "1 Star",
-        "2 Stars",
-        "3 Stars",
-        "4 Stars",
-        "5 Stars"
-      ],
+      ratingOptions: ["1 Star", "2 Stars", "3 Stars", "4 Stars", "5 Stars"],
```

Comment:
Unless you have plans to make this mutable this would be better declared as a Const in the render method. 
Generally speaking you should only put things in state that can be changed.